### ISSUE TITLE
Maybe Bookmarks and Notes should just be a single model?

### ISSUE THREAD

**Issue by davidrunger** [Chebotarev/BookBlur#1]
You have:

Bookmarks:
id, owner_id, book_id, paragraph

Notes:
id, owner_id, bookmark_id, body

It seems like this could just be consolidated into a single Bookmark model with a "body" column? The only reason that this wouldn't make sense, it seems to me, is if maybe somehow the "owner_id" of a note might be different than the "owner_id" of the bookmark?, like, if you can leave notes on other people's bookmarks. But it's not clear to me that this is possible (or desirable) in your vision for your app.

**Comment by Chebotarev** [comment]
I agree, I think the tables were separate in my original plans because I thought of bookmarks as not necessarily having an associated note. I guess in that case I can just leave the body blank :p Thanks again!
